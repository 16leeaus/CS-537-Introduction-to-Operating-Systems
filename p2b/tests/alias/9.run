./mysh tests/9.in | sort --ignore-case
