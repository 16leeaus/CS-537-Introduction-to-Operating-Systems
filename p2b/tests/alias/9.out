alias
alias
alias ls /bin/ls
alias whoami /usr/bin/whoami
exit
ls /bin/ls
ls /bin/ls
unalias whoami
whoami
whoami /usr/bin/whoami
